components/esp_modem/include/cxx_include/esp_modem_command_library.hpp

```diff
@@ -22,6 +22,14 @@ namespace dce_commands {
  * @{
```

Comment:
Unrelated `SPDX-FileCopyrightText: 2021-2022` -> `SPDX-FileCopyrightText: 2021-2023`

Reply:
Not sure why the pre-commit hook for copyrights didn't trigger. Let me check.